internal/operator/cluster/pgbouncer.go

```diff
@@ -483,12 +551,13 @@ func createPgBouncerDeployment(clientset kubernetes.Interface, cluster *crv1.Pgc
 
 	// get the fields that will be substituted in the pgBouncer template
 	fields := pgBouncerTemplateFields{
-		Name:            pgbouncerDeploymentName,
-		ClusterName:     cluster.Name,
-		CCPImagePrefix:  util.GetValueOrDefault(cluster.Spec.CCPImagePrefix, operator.Pgo.Cluster.CCPImagePrefix),
-		CCPImageTag:     cluster.Spec.CCPImageTag,
-		Port:            operator.Pgo.Cluster.Port,
-		PGBouncerSecret: util.GeneratePgBouncerSecretName(cluster.Name),
+		Name:               pgbouncerDeploymentName,
+		ClusterName:        cluster.Name,
+		CCPImagePrefix:     util.GetValueOrDefault(cluster.Spec.CCPImagePrefix, operator.Pgo.Cluster.CCPImagePrefix),
+		CCPImageTag:        cluster.Spec.CCPImageTag,
+		Port:               operator.Pgo.Cluster.Port,
```

Comment:
Thinking the port should be coming from the cluster spec here, and not `pgo.yaml` (e.g. in case `pgo.yaml` has changed since the cluster was created).

Reply:
Good catch. That may be a holdover from the original code, but I think it should be updated.

Reply:
This came up in https://github.com/CrunchyData/postgres-operator/issues/1830#issuecomment-684841322.

Reply:
Fixed.